pkgs/data/themes/obsidian2/default.nix

```diff
@@ -2,21 +2,17 @@
 
 stdenv.mkDerivation rec {
   name = "theme-obsidian2-${version}";
```

Comment:
Please change this to ```pname``` and drop the ```-${version}``` part.

Reply:
Done.